components/bin_packing_utils.py

```diff
@@ -161,23 +161,22 @@ def plot_cuboids(sample: dimod.SampleSet, vars: "Variables",
                       dz[i].energy(sample)))
     fig = _plot_cuboids(positions, sizes, bins.length * num_bins,
                         bins.width, bins.height, color_coded, cases.case_ids)
+    from itertools import product
+    corners = list(product([0, bins.length], [0, bins.width], [0, bins.height]))
+    ls = [bins.length, bins.width, bins.height]
     for i in range(num_bins):
-        fig.add_trace(
-            go.Scatter3d(x=[bins.length * i, bins.length * (i + 1)], y=[0, 0],
-                         z=[0, 0], mode='lines', name=f"Bin Boundary {i + 1}",
-                         line_color="red", line_width=5)
-        )
-        fig.add_trace(
-            go.Scatter3d(x=[bins.length * (i + 1)] * 2, y=[0, bins.width],
-                         z=[0, 0], mode='lines', name=f"Bin Boundary {i + 1}",
-                         line_color="red", line_width=5)
-        )
-        fig.add_trace(
-            go.Scatter3d(x=[bins.length * (i + 1)] * 2, y=[0, 0],
-                         z=[0, bins.height], mode='lines',
-                         name=f"Bin Boundary {i + 1}", line_color="red",
-                         line_width=5)
-        )
+        for c in corners:
```

Comment:
Suggest using more informative names for these intermediate variables like `corner` in place of `c`, etc.

Reply:
Actually, looked a bit more closely at this block, these are not corners of the bin but edges of the bin cuboid? Maybe a better name would be `edge` instead of `c` then 😄